packages/noya-state/src/selectors/layerSelectors.ts

```diff
@@ -64,3 +65,14 @@ export const makeGetPageLayers = (
       .map((id) => page.layers.find((layer) => layer.do_objectID === id))
       .filter((layer): layer is PageLayer => !!layer);
 };
+
+export const deleteLayers = (layers: IndexPath[], page: Sketch.Page) => {
+  layers.forEach((indexPath) => {
```

Comment:
Now that we have a separate `deleteLayers` function, maybe we can move the `.reverse` into here? Reversing is only needed for deletion anyway, so it'd be simpler if other functions didn't need to consider it.

Also, we can reverse the parameter order and rename to match other functions.

```ts
export const deleteLayers = (page: Sketch.Page, indexPaths: IndexPath[]) => {
  const reversed = [...indexPaths].reverse()
  reversed.forEach(...)
}
```

Reply:
I hope a re-order it correctly, because I was using the last value for the list in some part because it was reversed now It's the first value 🤔

Reply:
Oh yeah, that's true. But hopefully it will be simpler without reversing in the first place